src/components/layout/postForm/PostForm/index.tsx

```diff
@@ -0,0 +1,265 @@
+
+  // 주소 관련
+  const openAddressPopup = useDaumPostcodePopup(SCRIPT_URL);
+
+  const handleComplete = (data: { address: string }) => {
+    setValue('address', data.address);
+  };
+
+  const handleAddressButtonClick = () => {
+    openAddressPopup({
+      onComplete: handleComplete,
+      left: window.screen.width / 2 - 500 / 2,
+      top: window.screen.height / 2 - 600 / 2,
+      theme: addressCustomTheme,
+    });
+  };
+
+  // 스케줄 관련
+  const [scheduleArray, setScheduleArray] = useState<{ date: string; startTime: string; endTime: string }[]>([]);
+  const [isScheduleSelected, setIsScheduleSelected] = useState(false);
+  const selectedScheduleArray = getValues(['date', 'startTime', 'endTime']);
+  const watchDateField = watch('date');
+  const watchStartTimeField = watch('startTime');
+  const watchEndTimeField = watch('endTime');
+
+  useEffect(() => {
+    const convertedStartTime = convertTimeStringToNumber(watchStartTimeField);
+    const convertedEndTime = convertTimeStringToNumber(watchEndTimeField);
+    if (convertedStartTime !== undefined && convertedEndTime !== undefined) {
+      setIsScheduleSelected(watchDateField && convertedStartTime < convertedEndTime);
+    }
+  }, [watchDateField, watchStartTimeField, watchEndTimeField]);
+
+  const handleAddSchedules = () => {
+    const newSchedule = {
+      date: selectedScheduleArray[0],
+      startTime: selectedScheduleArray[1],
+      endTime: selectedScheduleArray[2],
+    };
+
+    const isDuplicatedSchedule = scheduleArray.some(
+      (schedule) =>
+        schedule.date === newSchedule.date &&
+        schedule.startTime === newSchedule.startTime &&
+        schedule.endTime === newSchedule.endTime,
+    );
+
+    if (!isDuplicatedSchedule && scheduleArray.length < 5) {
+      setScheduleArray([...scheduleArray, newSchedule]);
+    }
+  };
+
+  const handleRemoveSchedules = (index: number) => {
+    const newScheduleArray = [...scheduleArray];
+    newScheduleArray.splice(index, 1);
+    setScheduleArray(newScheduleArray);
+  };
+
+  // 이미지 관련
+  const imageArray: string[] = [];
+
+  const handleFilesAdd = (uploadedFiles: File[]) => {
+    const newImageArray: string[] = [];
+    uploadedFiles.forEach((file) => {
+      newImageArray.push(VALID_IMAGE_URL.usual + file.name);
+    });
+
+    imageArray.length = 0;
+    imageArray.push(...newImageArray);
+  };
+
+  // 등록 버튼 클릭 후 데이터 가공 관련
+  const handleEditFormData = () => {
+    const { title, price, address, headcount, description } = getValues();
+    const newBannerImageUrl =
+      imageArray.length === 0 ? VALID_IMAGE_URL.unusual + DEFAULT_API_DATA.bannerImageUrl : imageArray[0];
+    const newAddress = address === '' ? DEFAULT_API_DATA.address : address;
+    const titleArray = [category, title, price, newAddress, headcount];
+    const editedTitle = joinTitleByDelimiter(titleArray);
+
+    const editedRequestBody = {
+      title: editedTitle,
+      category: category,
+      description: description,
```

Comment:
키와 값이 동일하면 한 번만 쓰는거 어떨까요~
```suggestion
      category,
      description,
```